src/libraries/Microsoft.PowerFx.Core/Localization/ExpressionLocalizationHelper.cs

```diff
@@ -32,7 +32,7 @@ internal static string ConvertExpression(string expressionText, RecordType param
                 formula.ParseTree,
                 resolver,
                 config,
-                ruleScope: parameters._type,
+                ruleScope: parameters?._type,
                 updateDisplayNames: toDisplay,
```

Comment:
Should we be passing parameters at all here anymore?
If it's all been moved to resolver, this could be cleaned up.

Reply:
future pr?

Reply:
sure.